gulpfile.js

```diff
@@ -102,9 +110,25 @@ gulp.task("js:vendors", function() {
 /**
  * Server task
  */
+// gulp.task("server", function() {
+//    return gulp.src(opt.outputFolder)
+//     .pipe(webserver(opt.server));
+// });
+//
+var connect = require('connect');
+var livereload = require('connect-livereload');
+var staticFile = require('serve-static');
+var lrServer = require('tiny-lr')();
+
 gulp.task("server", function() {
-   return gulp.src(opt.outputFolder)
-    .pipe(webserver(opt.server));
+
+  lrServer
+    .listen(opt.server.livereload);
+
+  connect()
+    .use(livereload({ port: opt.server.livereload }))
```

Comment:
Nit: no space after opening curly bracket, no space before a closing one. Maybe we should add an eslint rule for this.